web-client/public/js/setup-load-and-import-handlers.js

```diff
@@ -51,6 +58,8 @@ const uploadHandler = (uploadRoute, uploader) => {
 };
 
 const networkErrorDisplayer = xhr => {
+    // re-enable upload button
+    disableUpload(false);
```

Comment:
Small naming note here: see how `disableUpload` as a function name doesn't line up with its parameter of true or false. ("disable upload to true" doesn't make a lot of plain English sense)

Instead, you can do this:
- `enableUpload()`
- `disableUpload()`
- `setUploadState(state)`